README.md

```diff
@@ -1,5 +1,24 @@
 # rbxlx-to-rojo
 Tool to convert existing Roblox games into Rojo projects by reading their `rbxlx` place files.
 
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need at least Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Let's assume we have a rbxlx file named "game", with a script in ServerScriptService.
```

Comment:
Would rather show how to do this in the tutorial, rather than just assuming the user knows how to do it.

Reply:
Lemme fix that.

Reply:
Might have fixed that. Check the latest commit.